assets/pages/bundles-landing/helpers/subscriptionsLinks.js

```diff
@@ -23,29 +29,20 @@ const defaultPromos: PromoCodes = {
   paperDig: 'p/GXX83X',
 };
 
-const tests = [
-  {
-    name: 'baseline',
-    intCmps: [
-      'gdnwb_copts_memco_sandc_support_baseline_support_banner',
-      'gdnwb_copts_memco_sandc_support_baseline_support_epic',
-      'gdnwb_copts_memco_sandc_support_baseline_support_liveblog',
-      'gdnwb_copts_memco_sandc_support_baseline_support_header_become_supporter',
-      'gdnwb_copts_memco_sandc_support_baseline_support_header_subscribe',
-      'gdnwb_copts_memco_sandc_support_baseline_support_side_menu_become_supporter',
-      'gdnwb_copts_memco_sandc_support_baseline_support_side_menu_subscribe',
-    ],
-    promoCodes: {
-      digital: 'p/DJ6CCJZLL',
-      paper: 'p/NJ6CCQLD4',
-      paperDig: 'p/NJ6CCSVMC',
-    },
+const customPromos : {
+  [Campaign]: PromoCodes,
+} = {
+  baseline_test: {
+    digital: 'p/DJ6CCJZLL',
+    paper: 'p/NJ6CCQLD4',
+    paperDig: 'p/NJ6CCSVMC',
```

Comment:
In order to have everything which is related to campaign codes in one place, we can probably move this inside the `guTracking` folder.